yarn-project/aztec-nr/value-note/Nargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "value_note"
 authors = ["aztec-labs"]
-compiler_version = "0.7.1"
+compiler_version = "0.18.0"
```

Comment:
This is because of the new semver check -- we could make it ">=0.18.0" or just remove it altogether because this will need to be changed once we move to 0.19 -- Theres an issue open to only supply this in the root Nargo.toml that would make this a lot less tedious

Reply:
Ok ok, did that in https://github.com/AztecProtocol/aztec-packages/pull/3082/commits/e20f81e310631b29fd3e940518e01778b0752b6c